setup.py

```diff
@@ -93,7 +94,7 @@ def run(self):
           packages=find_packages(exclude=['build.*', 'tests', 'tests.*']),
           install_requires=required,
           tests_require=test_required,
-          package_data={'coalib': ['default_coafile', 'VERSION',
+          package_data={'coalib': [Constants.system_coafile, 'VERSION',
                                    'bearlib/languages/documentation/*.coalang']
                         },
```

Comment:
@Makman2 can we do this, this way one more file needs to be free of external deps and python2 friendly!